otx/algorithms/detection/adapters/mmdet/utils/config_utils.py

```diff
@@ -264,16 +263,6 @@ def patch_evaluation(config: Config):
     config.early_stop_metric = "mAP"
 
 
-# TODO Replace this with function in common
-def patch_data_pipeline(config: Config, template_file_path: str):
-    """Update data_pipeline configs."""
-    base_dir = os.path.abspath(os.path.dirname(template_file_path))
-    data_pipeline_path = os.path.join(base_dir, "data_pipeline.py")
-    if os.path.exists(data_pipeline_path):
-        data_pipeline_cfg = Config.fromfile(data_pipeline_path)
-        config.merge_from_dict(data_pipeline_cfg)
-
-
```

Comment:
why should this be relocated to `utils.py`?

Reply:
The function used only in `detection` has been moved to `common` utils so that it can be commonly used in all tasks.